packages/snfoundry/contracts/Scarb.toml

```diff
@@ -6,9 +6,13 @@ edition = "2023_11"
 # See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
 
 [dependencies]
-starknet = "2.6.4"
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.14.0" }
-snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry", tag = "v0.27.0" }
+starknet = "2.8.2"
+openzeppelin = "0.17.0"
```

Comment:
Don't need to use full OZ, lets import the specific modules we need for this challenge

Reply:
Done! ✅